package-scripts/datadog-agent/postinst

```diff
@@ -39,8 +39,13 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
           configure)
               update-rc.d datadog-agent defaults
               update-rc.d datadog-agent enable >/dev/null 2>&1
-              adduser --system dd-agent --disabled-login --shell /bin/sh --no-create-home --quiet
-              usermod -d /opt/datadog-agent dd-agent
+              # Only add dd-agent user if it doesn't already exist
+              id -u dd-agent &>/dev/null
+              USER_EXISTS=$?
+              if [ ! $USER_EXISTS -eq 0 ]; then
+                  adduser --system dd-agent --disabled-login --shell /bin/sh --no-create-home --quiet
```

Comment:
I know there's not that many echoes in our `postinst` script but I think the creation of a new user on the machine is worth notifying. Could you just add a little something like `echo "Creating user 'dd-agent'."`  ?